### DISCUSSION TITLE
how to perform vulnerability statistics on certain agents

### DISCUSSION THREAD

**Discussion by linsonxie** [wazuh/wazuh#17119] [General]
The wazuh version I am currently using is 4.4.1, and I have a requirement to count the vulnerabilities of all or some agents. The existing APIs can only count individual agents, and I have found that the vulnerability data of agents is stored in /var/ossec/queue/db/{agent}.db The vuln_cves table, with one agent data and one database, makes it difficult to perform cross agent vulnerability statistics. Is there currently any way to achieve this requirement? In addition, agent and vuln_cves Is it supported to store in a MySQL or PostgreSQL database, as it currently exists in a sqlite database.

**Comment by eglyn** [comment]
I have the same request :)

I need to have a report of CVE from all Windows Agents, and it seems we can only see one agent at time :/

Is there a way to, at least, generate a full report of cve for all agents ? or groups ? 

The current vulnerability reporting system is not really usable. If you have to see each agent individually...

Thx :)
